components/brave_vpn/brave_vpn_os_connection_api.h

```diff
@@ -41,24 +51,122 @@ class BraveVPNOSConnectionAPI {
   void AddObserver(Observer* observer);
   void RemoveObserver(Observer* observer);
 
+  void set_shared_url_loader_factory(
```

Comment:
pls use CamelCase, same for others here

Reply:
AFAIK, simple setter and getter use snake case in chromium project?